lib/src/main/java/com/diffplug/spotless/kotlin/KtfmtStep.java

```diff
@@ -33,18 +32,35 @@
 	// prevent direct instantiation
 	private KtfmtStep() {}
 
-	private static final String DEFAULT_VERSION = "0.19";
+	private static final String DEFAULT_VERSION = "0.21";
 	static final String NAME = "ktfmt";
 	static final String PACKAGE = "com.facebook";
 	static final String MAVEN_COORDINATE = PACKAGE + ":ktfmt:";
+	private static String insertedVersion = DEFAULT_VERSION;
```

Comment:
Spotless is used in multithreaded environments. Static mutable state like this is a no-go. You can add `String version` to the `KtFmt$State` to achieve the same functionality, but in a thread-safe way.

Reply:
Done!